source/payment_flow/index.html.md.erb

```diff
@@ -282,8 +282,7 @@ the `return_url`, your service should:
 
 * display an appropriate final page, hosted by your service
 
-The [__Making payments__](/making_payments/#making-payments) section contains
-more details about how to match your users to payments.
+Read more about [matching your users to payments](https://docs.payments.service.gov.uk/making_payments/#choose-the-return-url-and-match-your-users-to-payments).
```

Comment:
Make this a relative link

Reply:
Thanks @jonathanglassman - fixed in https://github.com/alphagov/pay-tech-docs/pull/357/commits/31890ba88a10caa6ebc6f858308be8c7f81be604.